runtime/oti/vmargs_api.h

```diff
@@ -89,13 +89,14 @@ getOptionString(J9VMInitArgs* j9vm_args, IDATA index);
 	GET_COMPOUND returns all the option values from the command-line for a given option, separated by commas. Based on certain rules.
 	GET_COMPOUND_OPTS returns all the option values from the command-line for a given option, separated by \0s. Based on certain rules.
 	GET_MAPPED_OPTION returns the value of a mapped option. Eg. if -Xsov1 maps to Xj91 then "-Xsov1:foo" returns "foo"
-	GET_MEM_VALUE returns a rounded value of a memory option. Eg. "-Xfoo32k" returns (32 * 1024).
+	GET_RAW_MEM_VALUE returns the value of a memory option. Eg. "-Xfoo32k" return (32 * 1024).
+	GET_MEM_VALUE returns a pointer size rounded value of a memory option. Eg. "-Xfoo32k" returns (32 * 1024) and "-Xfoo1023" returns 1024.
```

Comment:
Please correct the spelling of "e.g." (at least for the part being changed).

This should provide an example showing how `GET_RAW_MEM_VALUE` differs from `GET_MEM_VALUE`.